qiime2/sdk/usage.py

```diff
@@ -396,6 +413,17 @@ def _init_metadata_(self, ref, factory):
 
         return result
 
+    def _init_data_collection_(self, ref, collection_type, *records):
```

Comment:
Please remove the guard on https://github.com/qiime2/qiime2/pull/539/files#diff-50eb2f5e696918079827fa682a7f9fd0R398-R402